mods/xpanes/README.txt

```diff
@@ -16,3 +16,6 @@ xyz (CC BY-SA 3.0):
 
 Gambit (CC BY-SA 3.0):
   xpanes_bar.png
+
+paramat (MIT):
```

Comment:
License for textures is always CC BY-SA 3.0, please could you change it?
Also, for perfection, add my name in license.txt after line 35.